sdk/python/kfp/v2/compiler_cli_tests/test_data/pipeline_with_various_io_types.py

```diff
@@ -25,9 +25,10 @@
 - {name: input_2, type: Float}
 - {name: input_3, type: }
 - {name: input_4}
-- {name: input_5, type: GCSPath}
+- {name: input_5, type: Metrics}
 - {name: input_6, type: Datasets}
 - {name: input_7, type: Some arbitrary type}
+- {name: input_8, type: {GcsPath: {data_type: TSV}}}
```

Comment:
It seems like we're converting GcsPath to parameter, is it intended?

Reply:
Or are we making the decision based on the placeholder type being used?

Reply:
Sorry I was a bit of confused because `input8` appears in the runtime parameter section in the golden file. Now I realized it represents the uri of the importer instead of the actual artifact.

Thanks!